00-default/blender.rules

```diff
@@ -0,0 +1,2 @@
+# Blender - https://www.blender.org/
+{ "name": "blender", "type": "Heavy_CPU" }
```

Comment:
I'm unfamiliar with both blender and davinci, but can you check from a process manager that these apps aren't just using `ffmpeg` under the hood when it's rendering things?

Reply:
For blender, they're 3d modeling software just like Cinema4D. And while yes, they're using ffmpeg **only** for rendering videos, however they use `blender` process to render everything else

Reply:
and nope, davinci resolve uses it's own thing for rendering (not ffmpeg)